docs/troubleshooting.md

```diff
@@ -0,0 +1,199 @@
+Spec:
+  Auth Info:
+    Bearer:
+      Secret Ref:
+        Kind:           Secret
+        Name:           ansibleservicebroker-client
+        Namespace:      ansible-service-broker
+  Ca Bundle:            LS0t...
+  Relist Behavior:      Duration
+  Relist Duration:      15m0s
+  Relist Requests:      0
+  URL:                  https://asb.ansible-service-broker.svc:1338/ansible-service-broker/
+Status:
+  Conditions:
+    Last Transition Time:       2017-10-05T17:22:01Z
+    Message:                    Error fetching catalog. Error getting broker catalog for broker "ansible-service-broker": Status: 401; ErrorMessage: <nil>; Description: invalid credentials, corrupt header; ResponseError: <nil>
+    Reason:                     ErrorFetchingCatalog
+    Status:                     False
+    Type:                       Ready
+  Operation Start Time:         2017-10-05T17:22:02Z
+  Reconciled Generation:        0
+Events:
+  FirstSeen     LastSeen        Count   From                                    SubObjectPath   Type            Reason                  Message
+  ---------     --------        -----   ----                                    -------------   --------        ------                  -------
+  38m           38m             1       service-catalog-controller-manager                      Warning         ErrorFetchingCatalog    Error getting broker catalog for broker "ansible-service-broker": Get https://asb.ansible-service-broker.svc:1338/ansible-service-broker/v2/catalog: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
+  38m           10m             107     service-catalog-controller-manager                      Warning         ErrorFetchingCatalog    Error getting broker catalog for broker "ansible-service-broker": Get https://asb.ansible-service-broker.svc:1338/ansible-service-broker/v2/catalog: x509: certificate signed by unknown authority
+  10m           17s             34      service-catalog-controller-manager                      Warning         ErrorFetchingCatalog    Error getting broker catalog for broker "ansible-service-broker": Status: 401; ErrorMessage: <nil>; Description: invalid credentials, corrupt header; ResponseError: <nil>
+```
+
+#### Resolution: Disable basic auth
+
+What you may notice in the output of `oc describe servicebroker` is that the service-catalog is being configured to use token based authentication
+to communicate with the Ansible Service Broker. However, because of [this bug](https://bugzilla.redhat.com/show_bug.cgi?id=1498992), the broker
+is configured to use basic auth by default. All that we need to do is 1) update the broker's configmap and 2) redeploy the broker.
+
+##### Update Broker's ConfigMap
+
+We need to update the Broker's configuration to disable basic auth. Use:
+
+```
+$ oc edit configmap broker-config
+```
+
+Modifying the configuration like what you see below:
```

Comment:
Disable the basic auth by setting the `enabled:` entry to `false` like the example below: